src/Containers/SearchFilterContainer.php

```diff
@@ -14,6 +14,12 @@ class SearchFilterContainer
 {
     public function call(Twig $twig, SearchService $searchService):string
     {
+        if (!$searchService->isAliveTest()) {
```

Comment:
I have found that the Internal Server Error was thrown, because in _SearchFilterContainer.php_, _$searchResults_ was null. 
So removed the check in this class, and added check for null in _SearchFilterContainer.php_.